CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Upcoming
 
+### Features
```

Comment:
I prefer having deprecations front and center at the top of the changelog for easier reference

Reply:
I was not aware on what you preferred. You want this after deprecations? Your way seems to be numpy way btw:

https://numpy.org/doc/stable/release.html

Reply:
Yes, deprecations first makes sense to me

Reply:
Done.